### DISCUSSION TITLE
Come ottenere lo stato di invio di una fattura elettronica?

### DISCUSSION THREAD

**Discussion by tobiascapin** [fattureincloud/api#394] [Q&A] [answered]
Buongiorno, sono riuscito a creare e inviare la fattura elettronica tramite API, ma non mi è chiaro come poter capire se una fattura è già stata inviata allo SDI ed eventualmente quel è il suo stato.
Come si deve fare?

**Comment by valmoz** [comment, marked as answer]
Buongiorno @tobiascapin ,
ha già controllato questa guida? [E-Invoice Management](https://developers.fattureincloud.it/docs/guides/e-invoice-management#mailbox_with_mail-send-the-e-invoice)

Grazie
Mauro

**Reply by valmoz** [reply]
Buongiorno @tobiascapin è riuscito a recuperare le informazioni desiderate?

Grazie
Mauro
